components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -352,7 +352,7 @@ function getPageBackground(p: HasImageProps) {
         background-attachment: fixed;
       `};
       background-position: center center;
-      background: ${getBackground};
```

Comment:
Setting `background` resets all the other `background` subproperties :facepalm: Just so happens the background image I've been using in development looks fine without a position/repeat/center :laughing: